app/scenes/Login/LoginView.js

```diff
@@ -0,0 +1,51 @@
+import {
+  View,
+  Text,
+  Image,
+  TextInput,
+  ScrollView
+} from 'react-native';
+import React, {PropTypes} from 'react';
```

Comment:
After React 15, proptypes has been moved to different package 

`prop-type`

lets import from it.